cmd/admin-rpc-server.go

```diff
@@ -209,7 +210,8 @@ func (s *adminCmd) CommitConfig(cArgs *CommitConfigArgs, cReply *CommitConfigRep
 	tmpConfigFile := filepath.Join(getConfigDir(), cArgs.FileName)
 
 	err := os.Rename(tmpConfigFile, configFile)
-	errorIf(err, fmt.Sprintf("Failed to rename %s to %s", tmpConfigFile, configFile))
+	ctx := logger.SetReqInfo(context.Background(), (&logger.ReqInfo{}).AppendTags("tmpConfigFile", tmpConfigFile).AppendTags("configFile", configFile))
```

Comment:
please split this into two lines... don't add AppendTags in the same line, it's too long to read or write it like this. 

```
reqinfo := (&logger.ReqInfo{}).AppendTags("tmpConfigFile", tmpConfigFile).
                                   AppendTags("configFile", configFile)
```

Reply:
modified it to 
```reqinfo := (&logger.ReqInfo{}).AppendTags("tmpConfigFile", tmpConfigFile)
reqInfo.AppendTags("configFile", configFile)
```